src/components/ClaimableBalances.tsx

```diff
@@ -53,6 +53,13 @@ export const ClaimableBalances = () => {
     return null;
   }
 
+  const getClaimBalanceHeader= () =>{
+    if(claimableBalances?.data.length===1){
+      return "Claimable Balance";
+    }
+    return "Claimable Balances";
+  };
```

Comment:
Better! Spacing is inconsistent in this block (Prettier should handle this automatically). Also, we could simplify it a bit more. Since the first word is the same in both cases, we could make it like this:
```
return `Claimable ${claimableBalances?.data.length === 1 ? "Balance" : "Balances"}`
```

What do you think?

Reply:
This is yet again a very elegant solution.  

I also tried with only adding the 's' 

return `Claimable Balance${claimableBalances?.data.length > 1 ? "s" : ""}`

But I feel it reduce readability. I added your suggestion.

Reply:
Thank you for the update! I agree that adding just an `s` makes the code more confusing.